releasetasks/templates/firefox/routes.yml.tmpl

```diff
@@ -0,0 +1,23 @@
+# Routing Templates: releases v2!
+
+# version (simple, direct route)
+{% do release_routes.append('releases.v2.{app}.version.{version}.build{build_num}'
+        .format(app=product, version=version.replace('.', '_'), build_num=buildNumber)) %}
+# pushdate
+{% do release_routes.append('releases.v2.{app}.pushdate.{year}.{mm}.{dd}.{version}.build{build_num}'
```

Comment:
Also in bug 1286075, instead of {year}.{mm}.{dd}, we now have {pushdate_long}, which is expanded into the appropriate date string. It might be worth considering the same thing here.

Though note that in the non-release routes, the pushdate_long is actually {year}.{mm}.{dd}.{pushdate}, where pushdate itself is {year}{mm}{dd}{HH}{MM}{SS}. So a pushdate of 2016/09/01 at 12:52:33 ends up as 2016.09.01.20160901125233

I'm not sure if it makes sense for release routes to have hour/minutes/seconds in them - if it does, we should probably use pushdate_long so that it matches non-release routes. If it doesn't make sense to do so, then we should probably stick with the {year}.{mm}.{dd} that you have.